addons/markers/ACE_Settings.hpp

```diff
@@ -0,0 +1,16 @@
+class ACE_Settings {
+    class GVAR(movableMarkersEnabled) {
+        category = CSTRING(Module_DisplayName);
+        value = 0;
+        typeName = "BOOL";
+        displayName = CSTRING(MovableMarkers_DisplayName);
+        description = CSTRING(MovableMarkers_Description);
+    };
+    class GVAR(moveRestriction) {
+        category = CSTRING(Module_DisplayName);
+        value = 0;
+        typeName = "SCALAR";
+        displayName = CSTRING(MoveRestriction_DisplayName);
+        description = CSTRING(MoveRestriction_Description);
```

Comment:
Should provide values, as such https://github.com/acemod/ACE3/blob/21502d7b7bc2516fdf3d674355d7ae1e892c65af/addons/missileguidance/ACE_Settings.hpp#L7